### DISCUSSION TITLE
Is the github pages for project-origin.github.io down ?

### DISCUSSION THREAD

**Discussion by wisbech** [project-origin/.github#25] [Documentation issues]
Hi awesome devs and ETT crew.

I have lately tried to access the github pages for project origin and I cannot acces them - is it part of the migration into smaller repositories ?
[https://project-origin.github.io/](https://project-origin.github.io/)

![image](https://github.com/project-origin/.github/assets/3016694/a05d74d6-e976-4672-b395-54cea3fdc811)

**Comment by MartinSchmidt** [comment]
you have to go to https://project-origin.github.io/docs/

**Reply by wisbech** [reply]
Nice - thank you
